### PR TITLE
Avi - issue #445 fix

### PR DESCRIPTION
**Pull request by ksivask**

### CODE REVIEW

File: repo/packages/A/avi/0/marathon.json.mustache [hunk 1]
```diff
@@ -60,6 +61,13 @@
         "hostPath": "/var/run/docker.sock",
         "containerPath": "/var/run/docker.sock",
         "mode": "RO"
+      },
+      {
+        "containerPath": "unused",
+        "mode": "RW",
+        "persistent": {
+          "size": 10
```
Comment (jdef) [inline repo/packages/A/avi/0/marathon.json.mustache]: is this disk truly "unused"? if so then why is it in this spec? if it's actually needed (and used to store data) then I'd recommend parameterizing the size here

**Comment by ksivask** [comment]
addressed PR review comments

File: repo/packages/A/avi/0/config.json [hunk 2]
```diff
@@ -101,6 +101,11 @@
               "minimum": 9161,
               "maximum": 9161
             },
+            "persistent-vol-size": {
+              "description": "Size of persistent volume",
+              "type": "integer",
+              "default": 10
```
Comment (jdef) [inline repo/packages/A/avi/0/config.json]: is there a max/min that makes sense for this?

**Comment by ksivask** [comment]
good point, yes min/max needed.

**Comment by ksivask** [comment]
helo James, I have made the changes as per the comments. Can you please review?

**Comment by pyronicide** [comment]
As you're adding functionality, please bump this package to a new version (1).

**Comment by ksivask** [comment]
more of a bug fix than functionality here. the persistence was something left out before 1.7live. 
For next set of features, I will bump the versions.

**Pull request merged**